src/components/Person/Person.jsx

```diff
@@ -1 +1,41 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  const partnerGender = sex === 'm'
+    ? 'wife'
+    : 'husband';
+
+  const personPartner = isMarried
+    ? `${partnerName} is my ${partnerGender}`
+    : 'I am not married';
+
+  const personAge = age
+    ? (
+      <p className="Person__age">
+        {`I am ${age}`}
+      </p>
+    )
+    : null;
```

Comment:
https://github.com/mate-academy/react_person/pull/1214#discussion_r1163816821
pay attention to the comments 🧑‍💻 
```suggestion
  const personAge = age && (
      <p className="Person__age">
        {`I am ${age}`}
      </p>
    );
```